test/test_fakeredis6.py

```diff
@@ -2106,6 +2115,156 @@ def test_zscore_wrong_type(r):
         r.zscore('foo', 'one')
 
 
+@pytest.mark.descriptor
+@testtools.run_test_if_redispy_ver("above", "4.0.0")
+def describe_zmscore() -> None:
```

Comment:
Can you move the hypothesis tests to `test_hypothesis.py`?

Reply:
`pytest-describe`-based Hypothesis tests replaced with simplified, individual, deterministic tests